scipy/linalg/lapack.py

```diff
@@ -48,6 +48,11 @@
 .. autosummary::
    :toctree: generated/
 
+   sorcsd
```

Comment:
Ideally these should show up in line 394 and similarly 685 for uncsd but somehow the latest additions didn't respect the alphabetical order apparently ☹️